lib/tasks/analyse/links.rb

```diff
@@ -132,18 +131,26 @@ def result_columns
       link_href
       number_of_tags
       taxon_base_paths
-    ].freeze
+    ]
   end
 
-  def add_results_to_worksheet(results, worksheet)
+  def format_columns(navigation_page_type)
```

Comment:
(I personally find this method name a little confusing. We're not actually formatting anything. What you're doing is returning the columns to be displayed for a given navigation page type. Maybe something like `columns_to_display`?)